### PR TITLE
Only validate OpenAPI doc files that have the versioned pattern in its name

### PR DESCRIPTION
**Pull request by va17**
* Only validate OpenAPI doc files that have the versioned pattern in its name

@rmpalomino @nalundgaard

### CODE REVIEW

File: scripts/validate_my_definition.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def validate_definition(definition_file):
     options = parser.parse_args()
     r = requests.get(OPENAPI_SCHEMA_URL)
     schema = r.json()
-    files = glob.glob(f"{options.dir}/*.yaml")
+    files = glob.glob(f"{options.dir}/*.v*.yaml")
```
Comment (nalundgaard) [inline scripts/validate_my_definition.py:46]: maybe `"{options.dir}/*.v[1-9]*.yaml"`?

Comment (nalundgaard) [reply scripts/validate_my_definition.py:46]: [fnmatch](https://docs.python.org/3.7/library/fnmatch.html#module-fnmatch) looks like it supports ranges, but not the regex repetition or real optional characters, but this should be at least more accurate.

Comment (rmpalomino) [reply scripts/validate_my_definition.py:46]: glob.glob calls fnmatch, so the range should work in the argument here too.

Comment (va17) [reply scripts/validate_my_definition.py]: Updated.
```python
>>> glob.glob("*.v[1-9]*.yaml")
['remediations.v1.yaml', 'assets_query.v1.yaml']
```

**Comment by nalundgaard** [comment]
@anton-b can you review and merge this?

**Pull request merged**